### PR TITLE
fix build without pg promise variables

### PR DESCRIPTION
**Pull request by BobanL**
# PULL REQUEST

## Summary
- Currently builds/images fail without any environment variables because PG-promise db connection is being made on the top level.
- Remove base .env since it just has local configuration
- Remove NEXT_PUBLIC_BASEPATH in favor of hard coding the value. [NEXT_PUBLIC](https://nextjs.org/docs/app/building-your-application/deploying#environment-variables:~:text=runtime%20environment%20variables.-,By%20default%2C%20environment%20variables%20are%20only%20available%20on%20the%20server.%20To,.,-You%20safely%20read) values are only modifiable during build time anyways so they aren't needed in this case.
- Add .dockerignore to improve local build speed

## Related Issue
Fixes #

## Acceptance Criteria
Please copy the acceptance criteria from your ticket and paste it here for your reviewer(s)

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/ecr-viewer/next.config.js [hunk 1]
```diff
@@ -20,10 +20,7 @@ const nextConfig = {
     ];
   },
   output: "standalone",
-  basePath:
-    process.env.NODE_ENV === "production"
-      ? process.env.NEXT_PUBLIC_BASEPATH
-      : "",
+  basePath: process.env.NODE_ENV === "production" ? "/ecr-viewer" : "",
```
Comment (austin-hall-skylight) [inline containers/ecr-viewer/next.config.js:23]: @emmastephenson do we think that STLT's are ever going to want to configure this path? I couldn't find any info in [the original ticket](https://github.com/CDCgov/phdi/issues/2536) about whether it should be configurable or not.

Comment (emmastephenson) [reply containers/ecr-viewer/next.config.js:23]: I think this should be fine from the STLT perspective; most of their networking changes will likely be before our custom paths. But would definitely double-check this with @rin-skylight and @alismx, they'll be most familiar with it.

Comment (austin-hall-skylight) [reply containers/ecr-viewer/next.config.js:23]: Ok, I've reviewed everything and am ready to approve as soon as we get an answer about this from devops.

Comment (rin-skylight) [reply containers/ecr-viewer/next.config.js:23]: The path reflected in this change is the one that we have pre-programmed in our App Gateway object, so I see no issue for use in Azure. I would defer a merge until @alismx affirmatively answers in regard to AWS.

Comment (alismx) [reply containers/ecr-viewer/next.config.js:23]: The value for `NEXT_PUBLIC_BASEPATH` defaults to `/ecr-viewer` in the AWS repos but can be overridden. If we're removing it here, I'd like to remove it there. From an AWS perspective, I see no problem with moving forward on this.

**Review by austin-hall-skylight — APPROVED** [review]

**Pull request merged**